### PR TITLE
Hijack #1676

### PR DESCRIPTION
**Pull request by sebweb3r**
Closes #1676

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2653,6 +2653,8 @@ athron->athlon
 athros->atheros
 atleast->at least
 atll->all
+atmoic->atomic
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:2656]: Maybe also `atmo->atom` and its plural? :shrug:

Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:2656]: I'm not sure about this one. There exist vlc, mozilla and Bosch plugins/products with this name.

Maybe Peter has to help us sorting here.

Comment (lurch) [reply codespell_lib/data/dictionary.txt:2656]: Fair enough, feel free to ignore :+1:

Comment (peternewman) [reply codespell_lib/data/dictionary.txt:2656]: Yeah people seem to use it to shorten atmosphere too. A quick random look at grep.app only found valid usage.

**Review by peternewman — APPROVED** [review]

**Comment by peternewman** [comment]
Nice, I didn't realise you could close a PR like this too!

**Pull request merged**